Extension/src/LanguageServer/client.ts

```diff
@@ -620,6 +627,26 @@ class DefaultClient implements Client {
         this.model.tagParserStatus.Value = notificationBody.status;
     }
 
+    private decoration : vscode.TextEditorDecorationType;
```

Comment:
Please move this up to the top of the class definition with the other member variables.  You might also want to rename it something more specific so we don't lose track of it.  e.g. inactiveRegionsDecoration.